ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master__okd-scos.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  ocp_4.16_base-rhel9:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+  origin_scos-4.16_base-stream9:
+    name: scos-4.16
+    namespace: origin
+    tag: base-stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+    origin_scos-4.16_base-stream9:
```

Comment:
Please use `images[].from` for the base image instead of an `inputs`. The reason being that if you change your Dockerfile's base image upstream, an `inputs` rule will stop applying for OKD and RHEL content will leak. Change the base image name in the upstream Dockerfile, however, will not affect `from: origin_scos-4.16_base-stream9`.

Reply:
updated